Application/Commands/Enquiry/Build/AddEnquiryCommand.cs

```diff
@@ -80,11 +80,11 @@ public async Task<SubmittedConfirmationModel> Handle(AddEnquiryCommand request,
 
         var enquiry = new Domain.Enquiry()
         {
-            SchoolId = request.Data?.SchoolId!,
-            Email = request.Data?.Email!,
-            TutoringLogistics = request.Data?.TutoringLogistics!,
-            SENDRequirements = request.Data?.SENDRequirements ?? null,
-            AdditionalInformation = request.Data?.AdditionalInformation ?? null,
+            SchoolId = request.Data!.SchoolId!,
+            Email = request.Data!.Email!,
+            TutoringLogistics = request.Data!.TutoringLogisticsDetailsModel.ToJson(),
+            SENDRequirements = request.Data!.SENDRequirements ?? null,
```

Comment:
The SENDRequirements and AdditionalInformation are nullable, so I suggest using the below.
```
SENDRequirements = request.Data?.SENDRequirements ?? null 
AdditionalInformation = request.Data?.AdditionalInformation ?? null
```
This approach ensures that the code handles the possibility of null values without triggering runtime errors.

Reply:
Have assumed that you're suggesting it should be:

```
            SENDRequirements = request.Data!.SENDRequirements,
            AdditionalInformation = request.Data!.AdditionalInformation,

```            
Which I have done.

Reply:
The approach I suggested above is the below:

```
SENDRequirements = request.Data?.SENDRequirements ?? null 
AdditionalInformation = request.Data?.AdditionalInformation ?? null
```

It ensures that the code handles the possibility of null values without triggering runtime errors.

Reply:
We don't need to use `request.Data?` since the earlier `ValidateRequest(request)` already checks that `request.Data` is not null.